Dockerfile

```diff
@@ -4,7 +4,7 @@
 # NOTES:            Currently this is one big dockerfile and non-optimal.
 
 
-ARG OPENSTUDIO_VERSION=2.6.1
+ARG OPENSTUDIO_VERSION=2.6.2
```

Comment:
We should rethink if we need SHAs on our dockhub tags....

Reply:
I don't think we wants shas when it is an official build (e.g. 2.6.1, 2.6.2). If we start full CD, then yes, we will probably want to do that.

Reply:
This doesn't directly answer the ? of SHAs, but i'm working on the CI planning now and i'm going to recommend we work toward more explicit versions ("2.6.2-pre") and corresponding docker tags, starting at OpenStudio and following the given release through all the related builds/tests.